scripts/minikube-minishift-setup-env.sh

```diff
@@ -7,15 +7,16 @@ shout() {
 }
 
 # Create a bin directory whereever script runs. This will be where all binaries that need to be in PATH will reside.
-mkdir bin artifacts
+mkdir -p go/bin artifacts
 # Change the default location of go's bin directory (without affecting GOPATH). This is where compiled binaries will end up by default
 # for eg go get ginkgo later on will produce ginkgo binary in GOBIN
-export GOBIN="`pwd`/bin"
+export GOPATH="`pwd`/go"
+export GOBIN="$GOPATH/bin"
 
 # Set kubeconfig to current dir. This ensures no clashes with other test runs
 export KUBECONFIG="`pwd`/config"
 export ARTIFACTS_DIR="`pwd`/artifacts"
-export CUSTOM_HOMEDIR=$ARTIFACT_DIR
+export HOME="`pwd`/home"
```

Comment:
Why not HOME=`pwd` ?

Reply:
Just want to make sure there is absolutely no chance of overlap. GOPATH, GOBIN and HOME